Dockerfile

```diff
@@ -1,13 +1,16 @@
-FROM python:3.9-slim
+FROM node:20.13.1-bookworm-slim
```

Comment:
just for the information and future work, would be nice to have consistent images across all repos.. other services we have node-slim and older versions.. for dev and prod its best if all of services use same versions to download and store less docker images